test/language/class/class-default-constructor-arguments.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class default constructor arguments
+---*/
+var args, self;
```

Comment:
Alternatively, you could place the assertions inside the constructor and then use a flag to ensure that codepath is hit (that might be slightly more consistent with the other tests)